### DISCUSSION TITLE
Ordering the language list Alphabetically

### DISCUSSION THREAD

**Discussion by hama73** [monkeytypegame/monkeytype#2775] [Ideas] [answered]
# Improving the language list order
While scrolling and navigating the languages in the settings or in the command line, I found it hard to detect which language comes later because they aren't ordered alphabetically, they are ordered just randomly.
Also, there's inconsistency from the command line's order to the setting's order.

### Here are 2 different examples of standard ordering of numerous language names
![10fastfinger.com](https://user-images.githubusercontent.com/96790752/160248301-ef61598d-4b2c-4776-9b3f-e913023aab17.png)
![Google Translate](https://user-images.githubusercontent.com/96790752/160248304-fdb1e870-4a44-4593-8899-e2351c0ed32e.png)

I think ordering the names Alphabetically would professionalize and make it easier to scroll down the languages list. Lastly, it would also make it easier and clearer for future contributor to know where to add their language's name in the list.

##### If you think this is a good idea, I can help with this and update the list order Alphabetically.

**Comment by ghost** [comment, marked as answer]
The list is currently ordered by popularity.
Alphabetical order is not desired, further information: #4189

**Reply by Miodec** [reply]
Both lists (commandline and settings) also offer a search bar.
